sw/host/opentitanlib/src/util/file.rs

```diff
@@ -130,17 +127,19 @@ mod tests {
             None,
             SockFlag::empty(),
         )?;
+        let snd = unsafe { OwnedFd::from_raw_fd(snd) };
+        let rcv = unsafe { OwnedFd::from_raw_fd(rcv) };
```

Comment:
Please could you add a `// SAFETY: ...` comment around these? (and below)

Reply:
These will just be temporarily here, and will be removed when we move from nix 0.26 to nix 0.27 or rustix which provides interfaces with native I/O safety.